contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.7.0--3.8.0.sql

```diff
@@ -9258,12 +9258,12 @@ CALL sys.babelfish_drop_deprecated_object('function', 'sys', 'sp_tables_internal
 -- Please have this be one of the last statements executed in this upgrade script.
 DROP PROCEDURE sys.babelfish_drop_deprecated_object(varchar, varchar, varchar);
 
+CREATE OR REPLACE PROCEDURE sys.sp_reset_connection()
+AS 'babelfishpg_tsql', 'sp_reset_connection_internal' LANGUAGE C;
+GRANT EXECUTE ON PROCEDURE sys.sp_reset_connection() TO PUBLIC;
```

Comment:
Please move this block above 
```
-- Drops the temporary procedure used by the upgrade script.
-- Please have this be one of the last statements executed in this upgrade script.
DROP PROCEDURE sys.babelfish_drop_deprecated_object(varchar, varchar, varchar);
```

Reply:
hmm does this affect anything. Just curious

Reply:
This will not affect anything but ideally we should put our SQL changes before dropping of this temporary procedure. Comment also mentions the same:
```
-- Please have this be one of the last statements executed in this upgrade script.
```
This is also done so that future changes don't miss `DROP PROCEDURE sys.babelfish_drop_deprecated_object(varchar, varchar, varchar);` in which case they won't be able to use this procedure